### ISSUE TITLE
Composes broken after v2.5.4

### ISSUE THREAD

**Issue by arggh** [nathantreid/meteor-css-modules#101]
I just updated to `Meteor@1.5-rc.6` and `css-modules@2.7.1`.

Upon initial startup, I get approximately 60 lines of this while CSS is being mangled: 

```
[TypeError: Cannot read property 'toString' of undefined]
[TypeError: Cannot read property 'toString' of undefined]
[TypeError: Cannot read property 'toString' of undefined]
...
```

Also, most of my CSS styles are missing. Weirdly, some are not. After initial startup, the error messages are gone, until I run `meteor reset`.

The project used to work fine `with Meteor@1.4.2.3` and `css-modules@2.3.1`.

**Comment by arggh** [comment]
Further study showed that the class names get properly generated and even the styles defined in CSS files work, but `composes` is broken. It doesn't matter if I use relative or absolute paths.

**Comment by arggh** [comment]
I did some more research and pinpointed the issue to have emerged at `v2.5.4`. Before that it seems to work fine, also with Meteor `1.5-rc.6`.

**Comment by akanix42** [comment]
Thanks, I'll take a look ASAP.

**Comment by akanix42** [comment]
I've been checking this in my test app, but can't reproduce it so far. I'll try it with the sample app tomorrow.
Could you post the contents of your `package.json`?
Thanks.

**Comment by arggh** [comment]
Sure! Here are the relevant bits:

```json
"devDependencies": {
    ...
    "autoprefixer": "^6.5.3",
    "node-sass": "^4.5.3",
    "postcss-color-function": "^3.0.0",
    "postcss-modules-extract-imports": "1.0.0",
    "postcss-modules-local-by-default": "1.0.0",
    "postcss-modules-scope": "1.0.0",
    "postcss-modules-values": "1.1.1",
    "postcss-nested": "1.0.0",
    "postcss-simple-vars": "1.1.0",
    "postcss-utilities": "^0.6.1"
  },
  "cssModules": {
    "ignorePaths": [
      "node_modules",
      ".meteor",
      "server",
      "lib",
      "packages"
    ],
    "globalVariables": [
      "client/styles/global/fonts.m.css",
      "client/styles/global/colors.m.css",
      "client/styles/global/variables.m.css"
    ],
    "extensions": [
      "m.css"
    ],
    "enableSassCompilation": [
      "m.css"
    ],
    "postcssPlugins": {
      "postcss-simple-vars": {
        "fileOptions": [
          "client/styles/global/variables.m.css"
        ]
      },
      "postcss-modules-values": {},
      "postcss-nested": {},
      "postcss-utilities": {},
      "postcss-color-function": {},
      "postcss-modules-local-by-default": {},
      "postcss-modules-extract-imports": {},
      "postcss-modules-scope": {}
    }
  },
  "postcss": {
    "plugins": {
      "autoprefixer": {
        ...
      }
    }
  }
...
```


And a sample of a CSS definition that failed:

```css
// file 1
.header {
  composes: pageHeader from '../../../styles/shared/typography.m.css';
}
```

```css
// file 2
.base {
  font-family: $headerFont;
  font-weight: 100;
}

.pageHeader {
  composes: base;
  text-align: center;
  font-size: 3rem;
}
```

**Comment by akanix42** [comment]
This was fixed in v2.7.2. It only occurred if the file being composed hadn't already been processed on it's own (so it was file load order dependent).

**Comment by arggh** [comment]
Thanks for the fix!
